snapcraft/_store.py

```diff
@@ -547,6 +548,14 @@ def history(snap_name, series, arch):
     print(tabulated_revisions)
 
 
+def latest_rev(snap_name, series=None, arch=None):
```

Comment:
being pythonic and all and this is not a direct reflection of a cli command, can you call this `get_latest_rev` and maybe add a docstring? Also document the return result.

Is there a change for there not to be a latest rev? Maybe some safe guards for that if necessary